README.md

```diff
@@ -140,6 +143,46 @@ interface VideoActions {
 </Video>
 ```
 
-# Author 🧔
+## Video text tracks 🚂
+
+> HTML5 [text tracks](https://html.spec.whatwg.org/multipage/media.html#the-track-element) support for videos.
+>
+> subtitles can be rendered natively, or they can be rendered using `VideoState.currentActiveCues` property:
```

Comment:
Is the `currentActiveCues` something that we need right now? As in, the existing use case requires a custom renderer? 

Otherwise I will defer that change for later just to reduce complexity

Reply:
I think the "wow" effect of our work will be to add a "metadata" track that reference our company resources while playing the video (subtitles/captions are cool, but not so impressive). For this track we need to "build" the feature in our player using `currentActiveCues`. Same for chapters actually...